src/Tribe/Views/V2/Assets.php

```diff
@@ -437,6 +437,20 @@ public function register() {
 				],
 			]
 		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-v2-block-editor-reskin',
+			'tribe-events-v2-block-editor-reskin.css',
+			[],
+			'wp_enqueue_scripts',
+			[
+				'priority'     => 10,
+				'conditionals' => [ $this, 'should_enqueue_frontend' ],
```

Comment:
We need a new check here on whether this is block editor or not. It should be combined with `'should_enqueue_single_event_styles'`, not `'should_enqueue_frontend'`

Reply:
I've updated this to `should_enqueue_single_event_styles`, I'm currently working on the check.

Reply:
UPDATE: I've added a new check i.e. `should_enqueue_block_editor_single_event_reskin_styles` to verify if we are on V2, on Event Single and if we are using the Block Editor. 

We can't use `should_enqueue_single_event_styles` because the function currently returns `false` if we're using the block editor.